docs/asciidoc/tcp-client.adoc

```diff
@@ -342,3 +343,70 @@ include::{examplesdir}/uds/Application.java[lines=18..33]
+ If the time to live of the DNS resource record returned by the DNS server is greater
+ than this max time to live, this resolver ignores the time to live from
+ the DNS server and uses use this max time to live.
+ Default to `Integer.MAX_VALUE`.
+| `cacheMinTimeToLive` | The min time to live of the cached DNS resource records (resolution: seconds).
+ If the time to live of the DNS resource record returned by the DNS server is less
+ than this min time to live, this resolver ignores the time to live from
+ the DNS server and uses this min time to live.
+ Default: 0.
+| `cacheNegativeTimeToLive` | The time to live of the cache for the failed DNS queries (resolution: seconds). Default: 0.
+| `disableOptionalRecord` | Disables the automatic inclusion of an optional record that tries to give a hint to the remote DNS server about
+ how much data the resolver can read per response. By default, this setting is enabled.
+| `disableRecursionDesired` | Specifies whether this resolver has to send a DNS query with the recursion desired (RD) flag set.
+ By default, this setting is enabled.
+| `maxPayloadSize` | Sets the capacity of the datagram packet buffer (in bytes). Default: 4096.
+| `maxQueriesPerResolve` | Sets the maximum allowed number of DNS queries to send when resolving a host name. Default: 16.
+| `ndots` | Sets the number of dots that must appear in a name before an initial absolute query is made.
+ Default: -1 (to determine the value from the OS on Unix or use a value of 1 otherwise).
+| `queryTimeout` | Sets the timeout of each DNS query performed by this resolver (resolution: milliseconds). Default: 5000.
+| `resolvedAddressTypes` | The list of the protocol families of the resolved address.
+| `roundRobinSelection` | Enables an
+ {nettyjavadoc}/io/netty/resolver/AddressResolverGroup.html[`AddressResolverGroup`] of
+ {nettyjavadoc}/io/netty/resolver/dns/DnsNameResolver.html[`DnsNameResolver`] that supports random selection
+ of destination addresses if multiple are provided by the nameserver.
+ See {nettyjavadoc}/io/netty/resolver/dns/RoundRobinDnsAddressResolverGroup.html[`RoundRobinDnsAddressResolverGroup`].
+ Default: {nettyjavadoc}/io/netty/resolver/dns/DnsAddressResolverGroup.html[`DnsAddressResolverGroup`]
+| `runOn` | Performs the communication with the DNS servers on the given
+ {javadoc}/reactor/netty/resources/LoopResources.html[`LoopResources`]. By default, the LoopResources
+ specified on the client level are used.
+| `searchDomains` | The list of search domains of the resolver.
+ By default, the effective search domain list is populated by using the system DNS search domains.
+| `trace` | A specific logger and log level to be used by this resolver when generating detailed trace
+ information in case of resolution failure.
+|=======
+
+Sometimes, you may want to switch to the JVM built-in resolver. To do so, you can configure the `TCP` client as follows:
```

Comment:
Same as for HTTP, put that first snippet before the table. Show how to access the configuration spec for `resolver` first, then list all the available options.